src/screens/components/profile/profile.jsx

```diff
@@ -35,6 +35,7 @@ const Profile = ({
             inline
           />
           <Field name="company" label="Company" type="text" component={input} inline />
+          <Field name="email" label="Email" type="text" component={input} inline />
```

Comment:
Can you put it between the field `fullname` and `company`?
Can you make it required (like the `fullname`)?

Reply:
```suggestion
          <Field name="email" label="Email" type="email" component={input} inline />
```

Surtout pour des questions de [clavier de saisie sur mobile](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) (et aussi pour la validation native)